src/cs/ui/InfoBar.cs

```diff
@@ -98,4 +104,20 @@ public partial class InfoBar : ProgressBar {
 	public void _HideInfo() {
 		Box.Hide();
 	}
+	
+	// The bar can have 2 colors, one when it's low and one normal
+	public void _UpdateColor(bool l) {
```

Comment:
What does the `l` parameter represent? I'm guessing it's a flag for when the tab is lowered. In any case it should be given a better name.